drivers/sensor/lsm6dso/lsm6dso.c

```diff
@@ -868,7 +886,6 @@ static int lsm6dso_init(const struct device *dev)
 
 #define LSM6DSO_SPI_OP  (SPI_WORD_SET(8) |				\
 			 SPI_OP_MODE_MASTER |				\
-			 SPI_LINES_SINGLE |				\
```

Comment:
I don't think this change is necessary. `SPI_LINES_SINGLE` still appears to be defined in `include/drivers/spi.h` and it is the only change in commit 6098434f0f8115262f61a94fb1d74ab1acb3f8c3. We can just get rid of this commit.